README.md

```diff
@@ -31,6 +31,13 @@ You are not required to install a binutils package, but it does speed up the ini
 3. Run `make` in the main directory.  
     **a.** Use the `-jN` argument to use `N` number of threads to speed up building. For example, if you have a system with 4 cores / 4 threads, you should do `make -j4`.
 
+### Building on MacOS
+1. `brew install gcc capstone coreutils`
+2. Put GNU-coreutils `bin` on path using by adding `PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"` to your shell rcfile (or update any GNU-util usage in `Makefile` and `tools/Makefile` to use the `g`-prefixed versions).
+3. Download latest `macos` build of `ido-static-recomp` from [here](https://github.com/decompals/ido-static-recomp/releases/), and add `cp` the binaries to `tools/ido-static-recomp/build5.3/out`.
+4. Take a look at [this gist](https://gist.github.com/tonyspumoni/fc1b4c9217c5f2821a9fc4cbf69b51ef) to see what other updates are needed to get the build to pass, and apply them if they are relevant to your environment. You may need to add specific `-I` and `-L` flags to get things like `openssl` and `gcc-12` includes/libs included.
```

Comment:
Currently, `openssl` is a requirement for dkr_assets_tool so might want to add this in step 1. Why is gcc-12 required?

Reply:
i'm not sure actually. I think brew install gcc installed as gcc-12 for me?

Reply:
FYI, I'm removing the `openssl` dependency in #269

Reply:
hell yeah